src/app/public/modules/shared/docs-tools-component-info.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Represents the structure of information returned from the SKY Supportal's `component-info` endpoint.
+ */
+export interface SkyDocsComponentInfo {
+  icon: string;
+  name: string;
+  summary: string;
+  url: string;
```

Comment:
I would make all of these properties optional (`?`) in case the API changes and we need to add/remove/adjust some of them.